app/views/dashboards/_dashboard_article.html.erb

```diff
@@ -24,7 +24,14 @@
   <a href="<%= article.current_state_path %>"><h2><%= "[Archived] " if article.archived %><%= article.title %></h2></a>
   <div class="dashboard-meta-details">
     <% if article.published %>
-      <%= article.readable_publish_date %>
+      <div class="dashboard-article-dates">
+        <span>
+          Published: <%= article.readable_publish_date %>
+        </span>
+        <span>
```

Comment:
this is going to add an empty span if there's no edit date, I think the check should be done outside of it